### ISSUE TITLE
IS macthing: delete unnecessary information of fows

### ISSUE THREAD

**Issue by Ecoxl** [Ecoxl/celero#67]

**Comment by tcgumus** [comment]
what are the unnecessary information?

**Comment by Xandalfius** [comment]
It's Probably about this "+" Option to show further Infos afaik.

![image](https://user-images.githubusercontent.com/69843619/97152168-d0b17980-1770-11eb-887c-9d306cad6d90.png)

**Comment by Ocimum-basilicum** [comment]
Yes exactly @Xandalfius. For me the question here is if we want to invest time in this old UI if @tcgumus is working on something better. I created an Issue for  https://github.com/Ecoxl/celero/issues/86
